Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM node:16-slim AS BUILD
-COPY . /tmp/src
 # install some dependencies needed for the build process
 RUN apt update && apt install -y build-essential make gcc g++ python3 ca-certificates libc-dev wget git
 
+COPY . /tmp/src
```

Comment:
Wasn't sure if this should go in it's own PR, I was going to revert it, but yeah, it does speed up building by quite a bit when trying to test changes on a remote machine instead of locally.